scripts/ci/run-benchmark

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+"""Script to benchmark several high level cli commands.
+
+As of now this benchmarks `cp` and `rm` with test cases for multiple 4kb files
+(default 10000 files) and a single large file (default 10gb, `cp` only).
+"""
+import os
+from subprocess import check_call
+from datetime import datetime
+import random
+import argparse
+
+TEST_BUCKET = os.environ.get('PERF_TEST_BUCKET')
+MANY_FILES_DIR = "files/10k"
```

Comment:
I think we should have a well defined place where we put both the local files we generate as well as the results dir.  Right now, this location depends on where you run the script.

I've been using `<reporoot>/workdir`, but I'm not tied to a particular name.  I do think we should say that everything related to running the perf scripts should use a single top level directory from the repo root regardless of what directory you're in when you run the script.